### PR TITLE
Adds Telegram binary 0.1.19

### PR DESCRIPTION
**Pull request by aktowns**
First package! :smile: 

I tried to go down the compile from source route, but it looks quite painful as telegram 
are using a patched qt5 and a bunch of other things :(

### CODE REVIEW

File: pkgs/applications/networking/instant-messengers/telegram/default.nix [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    icon = "Telegram";
+    type = "Application";
+    comment = "Telegram Instant Messenger";
+    desktopName = "Telegram";
+    genericName = "Instant Messenger";
+    categories = "Application;Network;";
+  };
+
+  meta = {
+    description = ''
```
Comment (jagajaga) [inline pkgs/applications/networking/instant-messengers/telegram/default.nix]: Please make description following our rules (https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes).

File: pkgs/applications/networking/instant-messengers/telegram/default.nix [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+, libxkbcommon, makeWrapper, libredirect, xkeyboard_config, makeDesktopItem }:
+
+stdenv.mkDerivation rec {
+  name = "telegram-${version}";
+
+  version = "0.9.10";
+
+  src = fetchurl {
+    urls = [
+      "https://updates.tdesktop.com/tlinux/tsetup.${version}.tar.xz"
```
Comment (jagajaga) [inline pkgs/applications/networking/instant-messengers/telegram/default.nix]: Is it the official site? I saw https://desktop.telegram.org/.

**Comment by jagajaga** [comment]
Please don't for get to properly name commit following our guide in `CONTRIBUTING.md`.

File: pkgs/applications/networking/instant-messengers/telegram/default.nix (hunk 2 above)
Comment (matthiasbeyer) [inline pkgs/applications/networking/instant-messengers/telegram/default.nix]: @jagajaga tdesktop.com reroutes to desktop.telegram.org, though I'd vote for the latter in the sources here.

**Comment by Denommus** [comment]
@aktowns I tried to use a package similar to this in my system, but I was enable to insert foreign accents in the client (like á, õ, ê). Did you have the same problem?

Also, do you intend on maintaining this?

**Comment by jagajaga** [comment]
Ping

**Comment by volhovM** [comment]
@aktowns are you going to resolve problems with tests failure?

**Comment by aktowns** [comment]
@Denommus From memory i didn't have issues, though i've moved off NixOS so i can't check, since i'm not running Nix won't be maintaining 
@volhovM going to close this, since i don't have somewhere to test it on

**Pull request closed without merging**